tox.ini

```diff
@@ -1,10 +1,12 @@
 [tox]
-envlist = py27, pypy
+envlist = py27, pypy, py34
 
 [testenv]
 deps = 
     pytest
     pytest-capturelog
+    tzlocal
+    configobj
```

Comment:
Right, but you might want to run the tests *before* installing (which, for example, is the case when developing or packaging in a clean environment).